From: John Johansen <john@jjmx.net>
Date: Mon, 23 May 2022 22:24:53 +0000
Subject: Merge parser/capability.h: add missing <cstdint> include

Without the change apparmor build fails on this week's gcc-13 snapshot as:

    capability.h:66:6: error: variable or field '__debug_capabilities' declared void
       66 | void __debug_capabilities(uint64_t capset, const char *name);
          |      ^~~~~~~~~~~~~~~~~~~~
    capability.h:66:27: error: 'uint64_t' was not declared in this scope
       66 | void __debug_capabilities(uint64_t capset, const char *name);
          |                           ^~~~~~~~
    capability.h:23:1: note: 'uint64_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
       22 | #include <linux/capability.h>
      +++ |+#include <cstdint>
       23 |

MR: https://gitlab.com/apparmor/apparmor/-/merge_requests/882
Approved-by: John Johansen <john@jjmx.net>
Merged-by: John Johansen <john@jjmx.net>
Signed-off-by: John Johansen <john.johansen@canonical.com>
---
 parser/capability.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/parser/capability.h b/parser/capability.h
index 3f65613..eed8f2c 100644
--- a/parser/capability.h
+++ b/parser/capability.h
@@ -19,6 +19,7 @@
 #ifndef __AA_CAPABILITY_H
 #define __AA_CAPABILITY_H
 
+#include <cstdint>
 #include <linux/capability.h>
 
 #define NO_BACKMAP_CAP 0xff
